src/styles/Weather.css

```diff
@@ -0,0 +1,28 @@
+.weatherContainer {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  background: rgba(255, 255, 255, 0.05);
+  box-shadow: 0 25px 45px rgba(0, 0, 0, 0.2);
```

Comment:
you can make these into custom variables! like `background: var(--color)` or `box-shadow: var(--shadow)`